content/blog/hpe-athonet-llm-platform-first-pillar-from-personal-assistant-to-collaborative-corporate-tool.md

```diff
@@ -0,0 +1,29 @@
+  - Private Networks
+  - 5G
+---
+Building on the insights from our previous blog “[The transformative impact of generative AI on Telco products](https://developer.hpe.com/blog/the-transformative-impact-of-generative-ai-on-telco-products/)”, this installment delves deeper into the foundational principles underpinning our strategy: specifically, the shift from utilizing isolated tools to embracing a holistic, team-centric collaborative framework.
+
+Sam Altman and OpenAI are on a mission to develop an exceptional personal assistant aimed at enhancing individual learning and support. While this initiative is great, we believe that transitioning from a business-to-consumer (B2C) to a business-to-business (B2B) model necessitates a significant shift in focus from serving individuals to empowering teams. So, we introduced the following **innovation of meaning**:
+
+![](/img/athon_col_tool.png)
+
+The rationale behind this shift is the conviction that teamwork is fundamental to organizational success. Our concern is that if this tool is used solely on an individual basis, it might inadvertently amplify personal biases, complicating communication within teams. By **emphasizing teamwork**, we aim to signal a broader perspective on how this technology should be utilized.
+
+To clarify, there are several global initiatives to address and reduce biases in LLMs, particularly those related to data, like the demographic’s ones. These biases can often be minimized or managed using LLMs themselves, employing strategies such as guardrails or reinforcement learning from human feedback methods. Adhering to the **HPE AI Ethics Principles**, we are also committed to mitigate these biases. However, we recognize that there is a distinct category of biases: the cognitive biases, which, unlike data biases, receive less focus in academic research and have fewer mitigation strategies available. Cognitive biases, such as confirmation bias, have profound effects on human interaction, potentially leading to polarization also within small social units like teams or departments. In response to this challenge, we are also dedicating efforts towards understanding and addressing cognitive biases. We are fostering environments that **encourage diverse perspectives by creating 'shared memory' spaces** within projects. In these spaces, employees can contribute with their individual chat memories, facilitating the collection of varied viewpoints and promoting openness and dialogue.
+
+In B2B environments, the transfer of employee knowledge into digital tools is crucial, underlining the significance of the design and presentation of these tools to the workforce. Historically, a common approach towards AI in business has been to automate processes and reduce costs, often at the expense of human resources. Our priority diverges from this path; we prioritize enhancing the intrinsic value of our products, focusing not just on reducing expenses but on adding qualitative benefits.
```

Comment:
"HPE's priority diverges from this path. Instead, it prioritizes enhancing the intrinsic value of its products..."